components/formats-bsd/src/loci/formats/in/FakeReader.java

```diff
@@ -1113,6 +1125,110 @@ private void fillRegions(MetadataStore store, int imageIndex) {
     }
   }
 
+  /**
+   * Translate key/value pairs from the INI table for the specified series.
+   */
+  private void parseSeriesTable(IniTable table, MetadataStore store, int newSeries) {
+    int s = getSeries();
+    setSeries(newSeries);
+
+    for (int i=0; i<getImageCount(); i++) {
+      String exposureTime = table.get("ExposureTime_" + i);
+      String exposureTimeUnit = table.get("ExposureTimeUnit_" + i);
+
+      if (exposureTime != null) {
+        try {
+          Double v = new Double(exposureTime);
+          Time exposure = FormatTools.getTime(v, exposureTimeUnit);
+          if (exposure != null) {
+            store.setPlaneExposureTime(exposure, newSeries, i);
+          }
+        }
+        catch (NumberFormatException e) {
+          LOGGER.trace("Could not parse ExposureTime for series #" + s + " plane #" + i, e);
+        }
+      }
+
+      String positionX = table.get("PositionX_" + i);
```

Comment:
We have 3 time more or less the same code block
It will be better to have a common method e.g. ``convert(String position, String positionUnit, int axis)`` where axis will indicate the method to use ``0= setPlanePositionX 1= setPlanePositionY, 2= setPlanePositionZ``

Reply:
Maybe with a TODO that with Java 8 just pass the method reference!